app/src/main/java/ingsw/group1/findmyphone/managing/Manager.java

```diff
@@ -97,14 +95,18 @@ public void sendAlarmRequest(SMSPeer smsPeer) {
      * @param phoneNumber    the number to which send your phone's location or active alarm.
      */
     public void analyzeRequest(String requestMessage, String phoneNumber) {
-        if (locationManager.isLocationRequest(requestMessage)) {
-            //Action to execute when device receives a Location request
-            sendResponseSms = new CommandResponseLocation(phoneNumber, currentContext.getApplicationContext());
-            locationManager.getLastLocation(currentContext.getApplicationContext(), sendResponseSms);
+        switch (MessageBuilder.getMessageType(requestMessage)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5068369)